frog/imports/api/sessions.js

```diff
@@ -117,6 +127,8 @@ Meteor.methods({
       graphId: copyGraphId,
       state: 'CREATED',
       timeInGraph: -1,
+      countdownStartTime: -1,
```

Comment:
why not use strings, like 'notStarted', 'paused'? Removes need to look it up. We will try to add Flow definitions for database collections too, where we can check this.